react/components/SelectionListOrderBy.js

```diff
@@ -18,19 +18,18 @@ const SelectionListOrderBy = ({
 }) => {
   const [showDropdown, setShowDropdown] = useState(false)
 
-  const handleDropdownBtClick = e => {
-    e.preventDefault()
+  const handleDropdownBtClick = useCallback(() => {
     setShowDropdown(!showDropdown)
-  }
+  }, [])
 
-  const handleOutsideClick = e => {
-    e.preventDefault()
+  const handleOutsideClick = useCallback(() => {
```

Comment:
You can simply `const handleOutsideClick = useCallback( () => setShowDropdown(false) )